source/guide_mattermost.rst

```diff
@@ -160,12 +171,94 @@ When upgrading to Mattermost 6.4 or newer you need to change the collation of th
   [isabell@stardust ~]$ mysql -e "ALTER DATABASE isabell_mattermost COLLATE = utf8mb4_general_ci;"
   [isabell@stardust ~]$
 
+Automatic Update
+-----------------
+
+Use the script attached to update Mattermost to the current version. Run the script above the mattermost-root directory.
+
+.. code-block:: shell
+
+	#!/bin/sh
+	printf "\n===\nWelcome to updating Mattermost... let us begin!\n===\n"
+
+	printf "\n== Preflight Checks ==\n"
```

Comment:
To increase the readability I would not use all this formatting stuff in all the info messages.
All these backslashes and other cosmetic characters make the code less readable.

If you really want this you can use a helper function for such things. You can even work with colors ... (different topic)
```bash
# This is what you are doing
function info_message { echo "\n== $* ==\n" }

# I think this is enough
function info_message { echo "INFO: $*" }

# usage
info_message "Stage 2 seperated"
```